### PR TITLE
AF-248 Updated submodule to generate new values of the enum

### PR DESCRIPTION
**Pull request by vova-nazarenko**

### CODE REVIEW

**Review by MartinBenn-AI — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,10 @@
 # Changelog
 All notable changes to this project will be documented in this file.
 
+## Version 6.15.0-AF-248 - 2024-10-10
```
Comment (AQI-DanG) [inline CHANGELOG.md:4]: Typically we do not add pre-release versions to the changelog and we definitely should not have pre-release versions with the same version number

**Review by AQI-DanG — CHANGES_REQUESTED** [review]
Please work with other team members to ensure we do not have any version conflicts

**Review by AQI-DanG — CHANGES_REQUESTED** [review]
I think we might need a quick call on this.  The changes that @josephquiroz  published as pre-release 6.15.0-pre-XX are actually breaking changes and should have been 7.0.0-pre-XX.  I think they are also not ready for publication in a non-pre-release version.  Because the enum changes are minimal my suggestion would be to publish those as the 6.15.0 release package.  We should also remove the current pre-release packages for that version and ensure that Joseph includes your changes in his publication of the 7.0.0 package.

@JonathanRobertson-AQI  and @MartinBenn-AI  - thoughts?

**Comment by AQI-DanG** [comment]
This should also contain a change to the version in the csproj file

**Pull request closed without merging**